src/components/ModalPayment/ModalCardDetails/ModalCardDetails.tsx

```diff
@@ -49,12 +52,17 @@ const ModalCardDetails = ({
   costPerMeal,
   nonProfitLocationId,
   campaignId,
```

Comment:
Do we need this? Especially when we can just do Campaign.Id and `campaignId` doesn't seem to be used anywhere

Reply:
`campaignId` is currently used as a prop in the regular GAM payment modal, so if we remove `campaignId` from the card modal, we'd have to update the regular GAM modal too. I do agree there's a bit of redundancy, so I think we can refactor after

Reply:
Also, one suggestion @justintmckibben had was to create different payment methods within a payment class depending on whether it's a donation, GAM or project (i.e. LUC) as opposed to having only one payment function. That would help ensure only the necessary props are ever passed